src/encoded/ingestion/ingestion_listener.py

```diff
@@ -0,0 +1,7 @@
+from snovault.ingestion import ingestion_listener as snovault_ingestion_listener
+from snovault.ingestion.ingestion_listener import IngestionListener, IngestionQueueManager, run  # noqa: F401 (imported but unused)
```

Comment:
Unused is not a sufficient explanation. It's a restatement of the problem you're trying to solve. Why are these needed? Are they re-exported? Or do they have a side-effect. We should perhaps add a `imported_for_side_effect(...)` in `misc_utils`.

As a matter of style, I also don't like putting noQA on an import that is not multi-line. There is far too much temptation for someone to add an extra import in the same line, tolerating the noQA, and then having "noQA creep". I recommend doing
```suggestion
from snovault.ingestion.ingestion_listener import (
    IngestionListener, IngestionQueueManager, run,  # noqa: F401 (imported but unused)
)
```
so that there is easy syntactic room for someone to put something on another line.

Also note that I added a comma after run so that the line containing the noQA doesn't have to be changed in order to add more elements.

Reply:
This file is no longer needed in cgap-portal; we share the one in snovault.